app/models/user.rb

```diff
@@ -7,6 +7,10 @@ class User < ApplicationRecord
   ##
   # Associations
   #
+  has_many :wissle
+  has_many :tag, :through => :interest
+  has_many :wissle, :through => :comment
+  has_many :wissle, :through => :subscription
```

Comment:
Plural

Reply:
Also, you cannot have more than one `has many :whatever`, since Rails infers the method names out of it. You should rename them and specify the class:

```
has_many :wissles
has_many :commented_wissles, :through => :comments, :class_name => 'Wissle'
has_many :subscribed_wissles, :through => :subscriptions, :class_name => 'Wissle'
```